types/postcss-less/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for postcss-less 3.1
+// Type definitions for postcss-less 4.0
 // Project: https://github.com/shellscape/postcss-less
 // Definitions by: Masafumi Koba <https://github.com/ybiquitous>
```

Comment:
@djcsdy Hi, thanks for the update. Can you add your ownership?

See also: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cc26a09ae59d6c1cdc0b2ed4e8a41aa075eafc1e/README.md#definition-owners

Reply:
Done.